recipes/cadishi/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "cadishi" %}
+{% set version = "1.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2e5e37416f90be2ae4bcd0bbb2d9ada1f7da1776256fef6efbaf6e5add8ed8d2
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - six
+    - future
+    - numpy
+    - scipy
+    - cython
+    - h5py
+    - pyyaml
+#    - wget
+#    - cudatoolkit-dev
```

Comment:
Please remove or commentr about when these will be needed

Reply:
Also the cudatoolkit-dev package is effectively dead.  @jakirkham - do we have docs about how to link against cuda yet?

Reply:
Not yet. Tracking this in issue ( https://github.com/conda-forge/conda-forge.github.io/issues/901 ).

There are however some recipes that people mention that are using this today. This could be [a good example]( https://github.com/conda-forge/nvidia-apex-feedstock ).

Reply:
I have removed these two commented lines in the latest commit.

The CUDA kernels are optional for Cadishi. I'm considering to submit a separate package 'cadishi-gpu' in the future.